src/libraries/System.Runtime/tests/ILLink.Descriptors.xml

```diff
@@ -1,3 +1,4 @@
 <linker>
   <assembly fullname="TestLoadAssembly" />
+  <assembly fullname="TestILAssembly" />
```

Comment:
Not sure if this is the best place to root the assembly in question, but it needs to be rooted to prevent failures in `EnableAggressiveTrimming` setups for FPTR testing.